includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -906,6 +906,7 @@ private function update_statement_descriptor( $statement_descriptor ) {
 	public function validate_account_statement_descriptor_field( $key, $value ) {
 		// Since the value is escaped, and we are saving in a place that does not require escaping, apply stripslashes.
 		$value = stripslashes( $value );
+		$value = trim( $value );
```

Comment:
I personally prefer to use one-liners in similar cases: it's simple core functions, so the meaning is straightforward and we have less code to get thru.